pkgs/misc/vim-plugins/overrides.nix

```
@@ -243,6 +243,15 @@ self: super: {
     dependencies = with super; [ webapi-vim ];
   });
 
+  minimap-vim = super.minimap-vim.overrideAttrs(old: {
+    preFixup = ''
+      substituteInPlace $out/share/vim-plugins/minimap-vim/plugin/minimap.vim --replace "code-minimap" \
+      "${code-minimap}/bin/code-minimap"
+      substituteInPlace $out/share/vim-plugins/minimap-vim/bin/minimap_generator.sh --replace "code-minimap" \
+      "${code-minimap}/bin/code-minimap"
```

```suggestion
      substituteInPlace $out/share/vim-plugins/minimap-vim/plugin/minimap.vim \
        --replace "code-minimap" "${code-minimap}/bin/code-minimap"
      substituteInPlace $out/share/vim-plugins/minimap-vim/bin/minimap_generator.sh \
        --replace "code-minimap" "${code-minimap}/bin/code-minimap"
```

We might want to use wrapProgram.